sdk/turing/router/config/experiment_config.py

```diff
@@ -32,7 +33,11 @@ def config(self, config):
         self._config = config
 
     def to_open_api(self) -> OpenApiModel:
+        kwargs = {}
+        if self.config is not None:
+            kwargs['config'] = self.config
+
         return turing.generated.models.ExperimentConfig(
             type=self.type,
-            config=self.config
+            **kwargs
```

Comment:
The autogenerated `ExperimentConfig` class doesn't take `None` for self.config, so I'm hiding away `config` into `kwargs`.

Reply:
Is there a reason why an empty dictionary would not work?

Reply:
That's a good point; I've just tried it out with an empty dictionary and it works perfectly!